tools/distpartitioning/convert_partition.py

```diff
@@ -320,34 +262,135 @@ def create_dgl_object(
             [int(type_nids[0]), int(type_nids[-1]) + 1]
         )
 
+    """
+    orig_nids = None
```

Comment:
what's this? if you do not need this piece of code, remove instead of comment out.